subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -42,7 +42,12 @@
 
 
 class run_volumes(SubSubtestCaller):
-    config_section = 'docker_cli/run_volumes'
+
+    def run_once(self):
+        ptc = self.config['pretest_cmd'].strip()
+        if ptc != '' and ptc is not None:
```

Comment:
AIUI the preferred idiom for this is just:

    if ptc:

(because null string and None are non-truthy). References available on request.